docs/readmes/cwf/dev_testing.md

```diff
@@ -0,0 +1,53 @@
+---
+id: dev_testing
+title: Test Carrier Wifi Access Gateway
```

Comment:
Up to you: title is pretty long, make it "Test CWAG" here (sidebar) and "Test Carrier Wifi..." below (actual page title)